src/common/responses/AuthorizationError.yml

```diff
@@ -1,14 +1,15 @@
   # This represents the body of a 401 error from the server.
   description: |
     Unauthorized. The error may indicate one of the following:
+
       * The `Authorization: Token` header is missing or malformed.
       * The API token value is missing from the header.
       * The token does not have sufficient permissions to write to this organization and bucket.
   content:
     application/json:
       schema:
-        $ref: "../schemas/Error.yml"
+        $ref: "../../common/schemas/Error.yml"
```

Comment:
Since we're in `common`, this should evaluate to the same place and the additional `../common/` is redundant